compiler/codegen.js

```diff
@@ -2086,12 +2086,20 @@ const generateCall = (scope, decl, _global, _name, unusedValue = false) => {
   let args = decl.arguments;
```

Comment:
Could we update this to be `[...decl.arguments]` just to avoid potential issues with altering the AST permanently?